googleapis-raw/Cargo.toml

```diff
@@ -30,8 +30,8 @@ exclude = ["grpc"]
 
 [dependencies]
 futures = "0.3.5"
-grpcio = "0.9.1"
-protobuf = "2.25.2"
+grpcio = "0.10.0"
+protobuf = "2.27.1"
```

Comment:
I missed updating these to match the Docker build since it pulls in this version so the change is now larger than I originally anticipated.